cmd/shipper-state-metrics/collector.go

```diff
@@ -337,7 +377,7 @@ func getNamespaces(lister kubelisters.NamespaceLister) ([]*corev1.Namespace, err
 		return nil, err
 	}
 
-	nsBlacklist := []string{"kube-system", "kube-public", "kube-dns", shipper.ShipperNamespace}
+	nsBlacklist := []string{"kube-system", "kube-public", "kube-dns", shipper.ShipperNamespace, shipper.GlobalRolloutBlockNamespace}
```

Comment:
I'd' suggest leaving a comment on the meaning of this group of namespaces: what do they define and what are the consequences if a namespace is mentioned here